data_structures/queue/test_queue.py

```diff
@@ -30,6 +39,11 @@ def test_insertion_small(small_queue):
     assert small_queue.back.val == 2
 
 
+def test_insertion_string(small_queue):
+    with pytest.raises(Exception):
```

Comment:
You should always test for specific exceptions to ensure that only the exception that you are expecting occurs.